source/run-commands.txt

```diff
@@ -0,0 +1,53 @@
+.. _run-commands:
+
+===========================
+Run Commands in ``mongosh``
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+For documentation of basic MongoDB operations in ``mongosh``, see:
+
+- :doc:`/crud`
+
+- :doc:`/run-agg-pipelines`
+
+- :doc:`/reference/methods`
+
+Terminate a Running Command
+---------------------------
+
+To terminate a running command or query in ``mongosh``,
+press ``Ctrl + C``.
+
+``Ctrl + C`` terminates the process in the shell, but does not
+terminate the process on the MongoDB server. This behavior
+differs from the :method:`db.killOp()` method, which terminates
+commands on the server.
+
+Command Exceptions
+------------------
+
+When running a helper method which wraps a database command (such as
+:method:`db.getCollectionNames`), if the server would return
+``{ ok: 0 }``, thereby indicating the method was not successful,
+``mongosh`` raises an exception.
+
+If you run the command using :method:`db.runCommand()` and the command
```

Comment:
Turns out this is actually not true - the driver will return an error even for `db.command`. If we're okay with it always throwing, then we should just update the docs to indicate that it will return an error in any case that `ok: 0`. If not OK then we have to make some code changes to make this statement true

Reply:
Tweaked the copy both here and in the changelog for the clarified behavior. Thanks!